docs/pages/admin-guides/deploy-a-cluster/helm-deployments/argocd-helm.mdx

```diff
@@ -93,9 +93,10 @@ ignoreDifferences:
 ```bash
 $ argocd app sync $YOUR_APPLICATION_NAME
 ```
-4. To verify setup, navigate to the 'Resources' page in your Teleport
-   cluster to confirm the Kubernetes cluster is registered.
-
+4. To verify setup check the readiness of the pods in the teleport namespace.
+```bash
+$ kubectl --namespace teleport get all
+```
```

Comment:
Our docs engine requires us to use the `code` block for copy-paste to work properly. Commands should reduce the output to the strict minimum (all -> pods) and if you are expecting the user to validate that something is working properly, it's better to show them an example of what they should see (the command output).

````suggestion
```code
$kubectl -n teleport get pods

NAME                                           READY   STATUS    RESTARTS   AGE
<your-agent-release>-0                         1/1     Running   0          14m
<your-agent-release>-1                         1/1     Running   0          14m
<your-agent-release>-updater-d9f97f5dd-v57g9   1/1     Running   0          16m
```
````